New Bylaws

ruthmarks151 opened the pull request:
These new bylaws have been created starting from Nova Scotia Alpha's Bylaws. These bylaws add laws for housing, and established bylaws for other aspirational programs like committees, attendance and academic accountability. I highly suggest you read them completely and eagerly await comments. After review by the chapter, advisors, and Alumni we will put it to a vote in chapter.

RyanMarks committed:
NSA Bylaws copied up to ARTICLE V markdown formatted

RyanMarks committed:
Completed formatting of NSA bylaws.

RyanMarks committed:
Converted Dal language to Mac, removed irrelevant bits

RyanMarks committed:
Amended Officer laws and Election Procedure to better reflect our chapter

RyanMarks committed:
Amended Article V to reflect our chapter's financial situation

RyanMarks committed:
Amended ARTICLE VI: MEMBER ACCOUNTABILITY

RyanMarks committed:
Reviewed remaining articles

RyanMarks committed:
Improved whitespace in the markdown file

RyanMarks committed:
Changes made in consult with Ben Logtenberg #253

RyanMarks committed:
Removed references to non-existent parking lot positions

RyanMarks committed:
Changes as suggested by Ben Sclodnick #254

RyanMarks committed:
Added appendix A and b

RyanMarks committed:
Updates inspired by existing bylaws and spellcheck

RyanMarks committed:
Added bylaw mandating membership contract

RyanMarks committed:
Addition of bylaw on parking spot management

RyanMarks committed:
Several smallish changes for consistency

Removed Exec’s ability to discipline
Cleaned up referencing within bylaws
Eased instant punishment for dress code
Gave all officers binders
Remove reimbursements for conferences
Removed a reference to executive officer committees 
Reordered committees

RyanMarks committed:
Changes to officer binders and academics

RyanMarks committed:
Amendments from Big Ideas chat

RyanMarks committed:
Added Brotherhood Chairman and his committee

RyanMarks committed:
Converted all 66%+1's to 66% for clarity

RyanMarks committed:
Added table of contents

RyanMarks committed:
Unbroke links

RyanMarks committed:
Renumbered bylaws and made ammendments

added Risk management committee definition
Cleaned language of several bylaws
Remove Membership transition Article, and moved 2 new bylaws to other sections

JemarJones reviewed (commented):
test comment for ryan

RyanMarks committed:
Changed header style, removed unused JC definition

RyanMarks committed:
Removed risk chair from auxiliary committees

RyanMarks committed:
Pro-privacy amendments to 33 suggested by Felipe

RyanMarks committed:
New initiation dues to reflect financial realities

RyanMarks committed:
Changes after another reading

Slight changed wordings and clarifications
Added Phikeia Pin+Book deposits
Added better committee descriptions

RyanMarks committed:
Changes for consistency, added house comm

RyanMarks committed:
Updates after Alumni commentary

- Changed officer review threshold to a majority of exec
- Changed language on drugs in housing to better reflect legal situations
- Added option for a IRV ballot to officer voting
- Reworded vacancy appointment section to better fit chapter realities and cover resignations
- Added 1 month time limit to reimbursements
-  Fixed typos
- Increased notice for special meeting absence to 3 days from 1
- Clarified 75
- Removed exec bylaw veto
- Replaced Member with Active Member when needed

Still outstanding
----------------
- Threshold of recruitment event attendance
- Resolve 4 years at mac vs 4 years as a Phi. See The Code Section 147, 174
- Article VIII and the RTA
- Definitions section?

RyanMarks committed:
Changes suggested by Phikeia

Grammar fixes
Improved clarity of 74
Increased restrictions on smoking
Cleared up academic requirement in elections

RyanMarks committed:
Guests are the responsibility of tenant and member

RyanMarks committed:
Changes based on chapter review
